src/protocol/utils.ts

```diff
@@ -72,16 +70,9 @@ function NotAllowed() {
 }
 
 export const DisallowEverythingProxyHandler: ProxyHandler<object> = {
-  // getPrototypeOf() {
-  //   NotAllowed();
-  //   return null;
-  // },
-  //has() {
-  //  NotAllowed();
-  //  return false;
-  //},
```

Comment:
As much as I hate to leave random commented-out code, I'd probably keep this as
```diff
-  // getPrototypeOf() {
-  //   NotAllowed();
-  //   return null;
-  // },
-  //has() {
-   //  NotAllowed();
-  //  return false;
-   //},
+  // getPrototypeOf() { },
+  // has() { },
```
Proxy handlers are a specific list of handlers and it's really nice to at a glance see which ones _aren't_ implemented.